src/main/java/edu/harvard/iq/dataverse/api/SavedSearches.java

```diff
@@ -46,22 +49,46 @@ public Response meta() {
     }
 
     @GET
+    @AuthRequired
     @Path("list")
-    public Response list() {
+    public Response list(@Context ContainerRequestContext crc) {
+
+        AuthenticatedUser superuser = null;
+        try {
+            superuser = getRequestAuthenticatedUserOrDie(crc);
+        } catch (WrappedResponse wr) {
+            return wr.getResponse();
+        }
+        if (superuser == null || !superuser.isSuperuser()) {
+            return error(Response.Status.UNAUTHORIZED, "Superusers only.");
+        }
+
         JsonArrayBuilder savedSearchesBuilder = Json.createArrayBuilder();
         List<SavedSearch> savedSearches = savedSearchSvc.findAll();
         for (SavedSearch savedSearch : savedSearches) {
             JsonObjectBuilder thisSavedSearch = toJson(savedSearch);
             savedSearchesBuilder.add(thisSavedSearch);
         }
         JsonObjectBuilder response = Json.createObjectBuilder();
-        response.add("saved searches", savedSearchesBuilder);
+        response.add("savedSearches", savedSearchesBuilder);
```

Comment:
Do we need this backward-incompatible change? What problem are we trying to solve?

Reply:
JSON Property names must be camelCased nothing more.